libs/sdk-model/src/execution/filter/index.ts

```diff
@@ -353,6 +353,15 @@ export function isRankingFilter(obj: unknown): obj is IRankingFilter {
     return !isEmpty(obj) && (obj as IRankingFilter).rankingFilter !== undefined;
 }
 
+/**
+ * Type guard checking whether the provided object is a filter.
+ *
+ * @public
+ */
+export function isFilter(obj: unknown): obj is IFilter {
+    return isDateFilter(obj) || isAttributeFilter(obj) || isMeasureValueFilter(obj) || isRankingFilter(obj);
+}
+
```

Comment:
please add tests for this guard. filter/tests/typeGuards.test.ts.. there are already a bunch of parameterized tests. you can probably base test isFilter on test for isRankingFilter... just copy-paste and switch excepted values

Reply:
fixed